docker/src/main/java/org/jclouds/docker/domain/ExecStartParams.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.docker.domain;
+
+import org.jclouds.json.SerializedNames;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * Json Parameter(s) (some of them) of Exec Start call.
+ */
+@AutoValue
+public abstract class ExecStartParams {
+
+   public abstract boolean detach();
+
+   @SerializedNames({ "Detach" })
+   public static ExecStartParams create(boolean detach) {
+      return builder().detach(detach).build();
+   }
+
+   public static Builder builder() {
+      return new AutoValue_ExecStartParams.Builder().detach(false);
+   }
+
+   @AutoValue.Builder
+   public abstract static class Builder {
+
+      public abstract Builder detach(boolean b);
+
+      public abstract ExecStartParams build();
+   }
```

Comment:
Some domain classes have builders, some not. Should we remove all the builders, given that these classes are pretty simple, for consistency?